neuromation/cli/job.py

```diff
@@ -853,6 +853,12 @@ def format_fail(job: str, reason: Exception) -> str:
         await browse_job(root, job)
 
     if not detach:
+        click.echo(
+            "This terminal is attached to the remote job, so you receive "
+            "the job output.\nUse 'Ctrl-C' to detach from this job (it "
```

Comment:
Could it be reformatted so that lines will break after `\n`?

Reply:
done, thanks